### PR TITLE
Fix `days` on `cci org import`

### PR DESCRIPTION
**Pull request by Br4nd0R**
# Changes
* `cci org import` now sets `days` to calculated value

# Issues Closed
* This closes issue #1101

### CODE REVIEW

File: cumulusci/cli/cci.py [hunk 1]
```diff
@@ -851,6 +856,15 @@ def org_import(config, username_or_alias, org_name):
     )
 
 
+def calculate_org_days(info):
+    """Returns the difference in days between created_date (ISO 8601),
+    and expiration_date (%Y-%m-%d)"""
+    created_date_str = info["created_date"].split("T")[0]
```
Comment (Br4nd0R) [inline cumulusci/cli/cci.py]: Not sure if there's a better way to easily convert the ISO datetieme to a simple date.

Comment (jofsky) [reply cumulusci/cli/cci.py]: datetime_format = "%Y-%m-%dT%H:%M:%S.%f+0000"
will work

Comment (Br4nd0R) [reply cumulusci/cli/cci.py]: well, I'm already using strptime() to process that date portion, which is all I want. I guess I was looking to see if there's a one-liner that converts the ISO formatted string to a date.

Comment (jofsky) [reply cumulusci/cli/cci.py]: Gotcha, not sure about that one.

**Pull request merged**